src/drivers/sensors/adxl345/adxl345.c

```diff
@@ -0,0 +1,91 @@
+/**
+ * @file adxl345.c
+ * @brief G-Sensor
+ * @author Denis Deryugin <deryugin.denis@gmail.com>
+ * @version
+ * @date 17.12.2019
+ */
+
+#include <stdint.h>
+
+#include <drivers/i2c/i2c.h>
+#include <drivers/sensors/adxl345.h>
+#include <framework/mod/options.h>
+#include <util/log.h>
+
+#define ADXL345_DEVID           0x00
+#define ADXL345_OFSX            0x1E
+#define ADXL345_OFSY            0x1F
+#define ADXL345_OFSZ            0x20
+#define ADXL345_OFS_AXIS(index) (ADXL345_REG_OFSX + (index))
+#define ADXL345_BW_RATE         0x2C
+#define ADXL345_POWER_CTL       0x2D
+# define ADXL345_POWER_CTL_MSR  (1 << 3)
+#define ADXL345_DATA_FORMAT     0x31
+#define ADXL345_DATAX0          0x32
+#define ADXL345_DATAY0          0x34
+#define ADXL345_DATAZ0          0x36
+#define ADXL345_DATA_AXIS(n)    (ADXL345_DATAX0 + (n))
+
+#define ADXL345_DEVID_VALUE     0xE5
+
+#define ADXL345_BUS    OPTION_GET(NUMBER, bus)
+#define ADXL345_OFFSET OPTION_GET(NUMBER, offset)
+
+struct adxl345_dev {
+	int i2c_bus;
+	int i2c_offset;
```

Comment:
Why do not name this field `i2c_addr`? I think addr is more common for i2s devices than offset.

Reply:
You're right. Changed it to `i2c_addr`